docs/user/openstack/deploy_sriov_workers.md

```diff
@@ -203,6 +203,25 @@ spec:
           configDrive: true
 ```
 
+If you are using Kuryr and your SR-IOV network has `port_security_enabled=False`, then you must create the worker node, setting its primary port with `portSecurity: false` and empty `spec.template.spec.providerSpec.value.securityGroups`. E.g.:
+
+```
+          networks:
+            - subnets:
+              - uuid: <machines_subnet_uuid>
+                portSecurityEnabled: false
+              portSecurityEnabled: false
+          securityGroups: []
+```
+
+In that case you can still apply the workers SG to the primary VM interface *after* the VM is created:
+
+```
+openstack port set --enable-port-security --security-group <infrastructure_ID>-<node_role> <main_port_id>
+```
+
+Those Kuryr specifics will be fixed in the future OpenShift versions.
```

Comment:
Is there a BZ to track the limitation? If not, could you please create one (and explicitly mentions reverting this PR) and reference it here?
We will also need a different BZ to include this documentation patch in the release-4.8 branch.

Reply:
There's https://bugzilla.redhat.com/show_bug.cgi?id=1964540, I added a note linking this patch there.